src/main/java/no/sikt/sws/IndexHandler.java

```diff
@@ -47,10 +49,15 @@ protected ProxyResponse<String> processProxyInput(
 
         try {
             var url = "/" + WorkspaceStripper.prefixUrl(resourceIdentifier, workspace);
+            var requestbody = body;
+
+            if ("_alias".equals(resourceIdentifier)) {
+                requestbody = WorkspaceStripper.prefixAliasBody(body, workspace);
```

Comment:
On line 52 you do requestbody = body, which is never used before its set to seomthing else.